test_shopcart.py

```diff
@@ -29,6 +29,175 @@ def test_index(self):
         self.assertTrue ('Shopcart Demo REST API Service' in resp.data)
 
 
+    def test_create_shopcarts(self):
```

Comment:
I think the test functions should be short, simple. 
For example, each block of the code can be one test function.